example/src/App.tsx

```diff
@@ -16,6 +16,7 @@ import { ComboWithGHScroll } from './release_tests/combo';
 import { TouchablesIndex, TouchableExample } from './release_tests/touchables';
 import Rows from './release_tests/rows';
 import Fling from './release_tests/fling';
+import Buttons from './release_tests/mouseButtons';
```

Comment:
```suggestion
import MouseButtons from './release_tests/mouseButtons';
```

I'd keep the full name, especially given that we support other buttons

Reply:
Done in [0cbaebd](https://github.com/software-mansion/react-native-gesture-handler/pull/2676/commits/0cbaebd85a445c9469aa45a6f9994c2d30d06578)